Sources/NIOHPACK/StaticTables.swift

```diff
@@ -0,0 +1,5015 @@
+//
+//===----------------------------------------------------------------------===//
+
+extension HeaderTableEntry {
+    fileprivate init(_ name: String, _ value: String?) {
+        self.init(name: name, value: value)
+    }
+}
+
+internal let StaticHeaderTable: [HeaderTableEntry] = [
+    HeaderTableEntry("", nil),              // real indexing begins at 1
+    HeaderTableEntry(":authority", nil),
+    HeaderTableEntry(":method", "GET"),
+    HeaderTableEntry(":method", "POST"),
+    HeaderTableEntry(":path", "/"),
+    HeaderTableEntry(":path", "/index.html"),
+    HeaderTableEntry(":scheme", "http"),
+    HeaderTableEntry(":scheme", "https"),
+    HeaderTableEntry(":status", "200"),
+    HeaderTableEntry(":status", "204"),
+    HeaderTableEntry(":status", "206"),
+    HeaderTableEntry(":status", "304"),
+    HeaderTableEntry(":status", "400"),
+    HeaderTableEntry(":status", "404"),
+    HeaderTableEntry(":status", "500"),
+    HeaderTableEntry("accept-charset", nil),
+    HeaderTableEntry("accept-encoding", "gzip, deflate"),
+    HeaderTableEntry("accept-language", nil),
+    HeaderTableEntry("accept-ranges", nil),
+    HeaderTableEntry("accept", nil),
+    HeaderTableEntry("access-control-allow-origin", nil),
+    HeaderTableEntry("age", nil),
+    HeaderTableEntry("allow", nil),
+    HeaderTableEntry("authorization", nil),
+    HeaderTableEntry("cache-control", nil),
+    HeaderTableEntry("content-disposition", nil),
+    HeaderTableEntry("content-encoding", nil),
+    HeaderTableEntry("content-language", nil),
+    HeaderTableEntry("content-length", nil),
+    HeaderTableEntry("content-location", nil),
+    HeaderTableEntry("content-range", nil),
+    HeaderTableEntry("content-type", nil),
+    HeaderTableEntry("cookie", nil),
+    HeaderTableEntry("date", nil),
+    HeaderTableEntry("etag", nil),
+    HeaderTableEntry("expect", nil),
+    HeaderTableEntry("expires", nil),
+    HeaderTableEntry("from", nil),
+    HeaderTableEntry("host", nil),
+    HeaderTableEntry("if-match", nil),
+    HeaderTableEntry("if-modified-since", nil),
+    HeaderTableEntry("if-none-match", nil),
+    HeaderTableEntry("if-range", nil),
+    HeaderTableEntry("if-unmodified-since", nil),
+    HeaderTableEntry("last-modified", nil),
+    HeaderTableEntry("link", nil),
+    HeaderTableEntry("location", nil),
+    HeaderTableEntry("max-forwards", nil),
+    HeaderTableEntry("proxy-authenticate", nil),
+    HeaderTableEntry("proxy-authorization", nil),
+    HeaderTableEntry("range", nil),
+    HeaderTableEntry("referer", nil),
+    HeaderTableEntry("refresh", nil),
+    HeaderTableEntry("retry-after", nil),
+    HeaderTableEntry("server", nil),
+    HeaderTableEntry("set-cookie", nil),
+    HeaderTableEntry("strict-transport-security", nil),
+    HeaderTableEntry("transfer-encoding", nil),
+    HeaderTableEntry("user-agent", nil),
+    HeaderTableEntry("vary", nil),
+    HeaderTableEntry("via", nil),
+    HeaderTableEntry("www-authenticate", nil)
+]
```

Comment:
Can we pull this out to a new file and just leave the Huffman table here? I really like the idea of sequestering the Huffman table as far from everything else as possible so that we never have to look at the damn thing. :wink:

Reply:
Most certainly.